app/views/cases/search_filters/_case_types.html.slim

```diff
@@ -0,0 +1,11 @@
+= form_for @query, url: search_cases_path, method: :get do |f|
+  = hidden_field_tag('search_query[parent_id]', @parent_id)
+  / = hidden_field_tag('search_query[filter_type]', 'case_type')
```

Comment:
Oh, hrm, I hand't thought of that as a requirement. I don't mind adding that back in so that we don't have to deduce from the params which filter tab was used.

Reply:
I'm not sure that's necessary.  They get the search results and all the tabs displayed in closed position, no?  So why do we need to know what tab they were last on?